cmake/Modules/FindTPLCUDA.cmake

```diff
@@ -10,8 +10,14 @@ IF (KOKKOS_CXX_COMPILER_ID STREQUAL Clang)
     ALLOW_SYSTEM_PATH_FALLBACK
    )
 ELSE()
-   KOKKOS_CREATE_IMPORTED_TPL(CUDA INTERFACE
-    LINK_LIBRARIES cuda
-   )
+   IF(WIN32)
+     KOKKOS_CREATE_IMPORTED_TPL(CUDA INTERFACE
+       LINK_LIBRARIES kokkoscore
```

Comment:
I don't get that one

Reply:
I haven't had a time to do a more extensive review yet - but I really think we need to discuss using FindCUDA or FindCUDAToolkit. I don't see any reason for us to re-engineer something built into CMake - particularly for Windows.

Reply:
See my comment above. This is not intended to be committed, I just didn't have a good solution right away.